web/components/requests/status/FcMenuStudyRequestsStatus.vue

```diff
@@ -0,0 +1,236 @@
+<template>
+  <v-menu
+    v-model="showMenu"
+    :close-on-content-click="false"
+    min-width="200">
+    <template v-slot:activator="{ on, attrs }">
+      <FcButton
+        :class="buttonClass"
+        :disabled="disabled"
+        type="secondary"
+        v-bind="attrs"
+        v-on="on">
+        <v-icon :color="status.color" left>mdi-circle-medium</v-icon>
+        <span>Status</span>
+        <v-icon right>mdi-menu-down</v-icon>
+      </FcButton>
+    </template>
+    <v-list>
+      <template v-for="(item, i) in items">
+        <v-list-item
+          v-if="item.items === undefined || item.items.length === 0"
+          :key="i"
+          class="fc-item-study-requests-status"
+          :disabled="item.disabled"
+          @click="actionMenu(item, null)">
+          <v-list-item-title>
+            <v-icon :color="item.value.color" left>mdi-circle-medium</v-icon>
+            <span>{{item.text}}</span>
+          </v-list-item-title>
+        </v-list-item>
+        <v-list-group
```

Comment:
right now, these groups (well, group, since there's only 1) stay open when the parent menu is closed.  we'd ideally like for them to close with the parent menu!

one way to do that is to keep a mapping from `item.value` to flags that determine whether each submenu is open - we can then add a watcher to set these flags to `false` whenever `showMenu` becomes `false`.